src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Resource/NewAzureResourceCmdlet.cs

```diff
@@ -76,12 +76,6 @@ public sealed class NewAzureResourceCmdlet : ResourceManipulationCmdletBase
         public Hashtable Tag { get; set; }
 
         /// <summary>
-        /// Gets or sets the zones.
-        /// </summary>
-        [Parameter(Mandatory = false, HelpMessage = "The zones.")]
-        public string[] Zones { get; set; }
```

Comment:
@TianoMS this is a breaking change; scripts using the `Zones` parameter for this cmdlet will no longer work